examples/nextjs-with-typescript/package.json

```diff
@@ -1,23 +1,28 @@
 {
   "name": "nextjs-with-typescript",
+  "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 3001",
+    "dev": "next dev -p 8087",
     "build": "next build",
-    "start": "next start -p 3001",
+    "start": "next start -p 8087",
     "lint": "next lint"
   },
   "dependencies": {
     "media-chrome": "link:../../",
-    "next": "12.0.10",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react": "link:../../node_modules/react",
```

Comment:
NOTE: using links to sidestep acrobatics/build errors that result from duplicate versions of react being pointed to. This is roughly analogous to complexities we encounter when doing a `yarn link` from "react flavored" elements for other projects such as stream.new. While we could alternatively rely on the [yarn file protocol](https://yarnpkg.com/protocol/file) for our media chrome dependency, this has the disadvantage of copying instead of symlinking, which means changes in media chrome won't get picked up without a dependency reinstall on the next example app.